### ISSUE TITLE
serial-by-id

### ISSUE THREAD

**Issue by sotechn** [feelfreelinux/octo4a#324]
Hi.
I'm trying to follow the instructions on the klipper website, it says that a call to ls /dev/serial-by-id/* is required.  on my ubuntu 22.04 this folder is empty, how can i access it?  I can use manual update, but I would like to be able to automatically download!  
Thanks.

**Comment by Kelsch** [comment]
Yeah I am in the same boat, but then I saw that this has a setup-klipper.sh script. What's that about and how do I use it in the app?

**Comment by Filo301** [comment]
Octo4a uses /dev/ttyOcto4a
Try ”sudo apt remove brltty” if you don't need it. It is software for "accessibility devices"
